tests/contrib/sensors/test_file_sensor.py

```diff
@@ -134,6 +135,56 @@ def test_default_fs_conn_id(self):
             task.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE,
                      ignore_ti_state=True)
 
+    def test_wildcard_file(self):
+        suffix = '.txt'
+        with tempfile.NamedTemporaryFile(suffix=suffix) as tmp:
+            fileglob = os.path.join(os.path.dirname(tmp.name), '*' + suffix)
+            task = FileSensor(
+                task_id='test',
+                filepath=fileglob,
+                fs_conn_id='fs_default',
+                dag=self.dag,
+                timeout=0,
+            )
+            task._hook = self.hook
+            task.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE,
+                     ignore_ti_state=True)
+
+    def test_subdirectory_not_empty(self):
+        suffix = '.txt'
+        dir_ = tempfile.mkdtemp()
```

Comment:
This will probably fail our pylint style checker (which is new since you opened the PR) so you'll need to change this when resolving conflicts/rebasing.
```suggestion
        dir = tempfile.mkdtemp()
```

Reply:
The pylint scripts in the CONTRIBUTING.md doesn't exists, so I'm not quite sure how do you guys run pylint, but by running it with this [pylintrc](https://github.com/apache/airflow/blob/master/pylintrc), I get the opposite.
As far as I know, according to PEP8, single leading underscore is the right way to avoid collisions with python builtin functions and keywords (in this case - `dir`).

Reply:
Cool if that passes then all's good. I just thought it _would_ fail. This is why we have computers check style and not humans 😁